.golangci.yml

```diff
@@ -0,0 +1,10 @@
+# See https://golangci-lint.run/usage/configuration/ for available options.
+linters:
+  enable:
```

Comment:
For this list, we could draw inspiration from the [Cilium .golangci.yaml file](https://github.com/cilium/cilium/blob/master/.golangci.yaml#L82-L92), although I would add `goimports` and `revive`.

Reply:
thanks for the pointer! updated the list. now running with:
```
INFO [lintersdb] Active 9 linters: [goerr113 goimports govet ineffassign misspell revive staticcheck unused varcheck]
```